docs/measurements/procedures/rigid_waist_shift.md

```diff
@@ -12,10 +12,68 @@
       After global corrections are done and trimmed in the machine, one applies a rigid waist shift in a given IR and scans the colinearity knob for the value that minimizes the $|C^{-}|$.
       These settings, when taking away the rigid waist shift, will minimize local coupling and its impact at the IP.
 
+## Rigid Waist Shift & LSA Knobs
+
+Some information about the rigid waist shift can be found in [this gallery page][rws_gallery]{target=_blank} from Felix's package's documentation.
+Some details about the creation of these knobs and their addition in LSA can be found in this [LHC OMC logbook entry](https://be-op-logbook.web.cern.ch/elogbook-server/GET/showEventInLogbook/3545713){target=_blank} and the ones that follow.
+
+### Knob Setting Convention
+
+Since the waist can be shifted in two directions - left and right of the IP - two knobs were generated per IP, for a unit setting of $1$ and -$1$.
+This unit setting is an arbitrary definition, where a value of $1$ corresponds to a $0.5$% change of the triplet powering.
+This, in turn, corresponds to  a waist shift of ~43-44cm in a given direction.
+
+The knob setting is defined as follows, with regards to Beam1 (reverse for Beam2):
+
+- Unit setting of +$1$: shifts the waist to the _left_ of the IP ($s_{waist}^{B1} < s_{IP}^{B1}$)
+- Unit setting of -$1$: shifts the waist to the _right_ of the IP ($s_{waist}^{B1} > s_{IP}^{B1}$)
+
+It is possible to generate knobs for different unit settings but for LHC operation these values are the ones that were used by Felix S.
+
+### The Knobs in LSA
+
+There are two different types of knobs that need to be trimmed in:
+
+- A knob for the triplet quadrupoles
+- A knob for the independent quadrupoles used to rematch the optics.
+
+A knob for the MQTs could be used to rematch the working point, but in operation the tune feedback should take care of this.
+Since the triplet knob for a unit setting of +$1$ is the opposite sign of the knob for a unit setting of -$1$, only a single triplet knob per IP has been added to LSA, which can then be trimmed with a factor of ±$1$.
```

Comment:
What? Are you telling me 1 has the opposite sign of -1? 
Where is your 360 pages of proof for that?

Reply:
This sounds like we only need one knob. I am now fully confused why there were two in the first place.

Reply:
Yes indeed, where's my PhD now please? More seriously, while it seems obvious it might not always be the case that the opposite knob corresponds to an opposite sign of all magnets in the knob, so I wanted to make that clear.

Reply:
> This sounds like we only need one knob. I am now fully confused why there were two in the first place.

See my answer to your first comment in the PR! You're right.

Reply:
Should be clearer after [14abe46](https://github.com/pylhc/pylhc.github.io/pull/74/commits/14abe462aac917a757b1b6b3d51852e686029608)